saleor/warehouse/models.py

```diff
@@ -25,8 +30,60 @@ def for_country(self, country: str):
             .order_by("pk")
         )
 
+    def applicable_for_click_and_collect_no_quantity_check(
```

Comment:
This function and the one below could also have some docstrings that would explain their assumptions or business logic used to resolve Warehouses. Ideally, there should be one function to do that, but we have two and it's not clear which should be used in which case.

Reply:
Done.